andela/northstar-backend
# #168024327 Create necessary database tables

### What does this PR do?
- Creates all the database tables needed in the application

### Description of tasks to be completed
Create the following tables
- users
- departments
- suppliers
- facilities
- likes
- rooms
- amenities
- feedbacks
- bookings
- ratings
- trips
- comments
- notifications

### How should this be manually tested?
-  Clone or download this repository
-  On your terminal, `cd` into the application's directory and run: `git checkout ch-database-168024327 `
-  Set the following environment variables in your `.env` file:
```
DEV_DB_USER // database username
DEV_DB_PASSWORD // database password
DEV_DATABASE  //name of your local database
DATABASE_URL  // production database url
```

- If you have not, install [Postgresql](https://www.postgresql.org/download)
- Run `npm run db:migrate`
- Check your database, you should now have all the previously listed tables in your database.
- To undo migration, run `npm run undo:migrate`

### What are the relevant Pivotal Tacker stories?
[#168024327](https://www.pivotaltracker.com/story/show/168024327)
